tests/test_pytomorrowio.py

```diff
@@ -132,25 +134,10 @@ async def test_timelines_hourly_good(aiohttp_client, mock_url):
     assert api.num_api_requests == 1
 
     assert res is not None
-    assert isinstance(res, Mapping)
```

Comment:
I think we should validate the shape of the response somewhere...

Reply:
We do? We check the list length and check the keys

Reply:
Like most tests, the reason to validate the shape is to ensure that future changes don't change it accidentally.